vector/src/main/res/values/strings.xml

```diff
@@ -489,7 +489,7 @@
     <string name="direct_chats_header">Conversations</string>
     <string name="template_local_address_book_header">${app_name} contacts</string>
     <string name="template_user_directory_header">${app_name} directory</string>
-    <string name="matrix_only_filter">Matrix contacts only</string>
+    <string name="template_matrix_only_filter">${app_name} contacts only</string>
```

Comment:
Can you please revert this change? the request #248 has been cancelled for the moment